src/components/list/List.tsx

```diff
@@ -11,7 +11,7 @@ type Props = {
 function List({ items, component: Component }: Props) {
   return (
     <ul className={styles.list}>
-      {items.map((item) => (
+      {items?.map((item) => (
```

Comment:
This was the changed line. I needed the add the ? there, without it the application was crashing on startup.